_includes/aip-header.html

```diff
@@ -9,7 +9,7 @@
             <a class="glue-header__logo-link" href="{{ '/' | relative_url }}" title="Google">
               <div class="glue-header__logo-container">
                 <svg role="img" aria-hidden="true" class="glue-header__logo-svg">
-                <use xlink:href="/assets/images/glue-icons.svg#google-color-logo"></use>
+                <use xlink:href="{{ '/assets/images/glue-icons.svg#google-color-logo' | relative_url }}"></use>
```

Comment:
A Jekyll "gotcha". If you do not have a CNAME defined, then you need to define a `baseurl` property in `_config.yml`. Then you can create relative urls using this method.

Would you be okay if I added this to the contributions guidelines? That all urls need to be in this format in order to support forks of aip.dev that don't have a CNAME?

Reply:
Sounds good to me.